Dockerfile

```diff
@@ -11,10 +11,9 @@ LABEL org.label-schema.name="greenkeeper-keeper" \
 
 ENV NODE_ENV=production
 
-RUN mkdir -p /app
-WORKDIR /app
-COPY . /app
+COPY . .
 EXPOSE 8080
-RUN npm install --prod
+RUN npm install --dev
+RUN npm run build
 
```

Comment:
worth adding a `RUN npm prune --production` after the build to prune the weight of the dev dependencies after they're used for the `build` step?